guards/utils/filter_manager/tests/test_filter_chain.py

```diff
@@ -0,0 +1,73 @@
+from guards.utils.filter_manager.filter_chain import FilterChain
+from guards.utils.filter_manager.filter_list import FilterList
+
+
+def test_filter_chain_instantiates():
```

Comment:
Classes can be used for grouping the tests for shorting the name of the test cases by removing the class name prefix (it is not that important but you can consider it), like 
- class `TestFilterChain` allows the tests below to skip test_`filter_chain` prefix,
- class `TestGuardManager` allows the tests below to skip test_`guard_manager` prefix,

Reply:
Done, refactored all the tests